### DISCUSSION TITLE
自己训练的nb模型嵌入Android项目后，APP打开出现闪退

### DISCUSSION THREAD

**Discussion by zx-lin-0825** [PaddlePaddle/PaddleOCR#14328] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

![1](https://github.com/user-attachments/assets/c80e927c-6421-4b71-979d-50fac134d31a)

### 🏃‍♂️ Environment (运行环境)

gradle-7.4-bin
compileSdkVersion 29
minSdkVersion 23
targetSdkVersion 29
versionCode 2
versionName "2.0"

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

[rec.zip](https://github.com/user-attachments/files/17841736/rec.zip)

**Comment by jingsongliujing** [comment]
建议您这边试试paddle x的端侧部署：https://paddlepaddle.github.io/PaddleX/latest/pipeline_deploy/edge_deploy.html

**Comment by zx-lin-0825** [comment]
> 建议您这边试试paddle x的端侧部署：https://paddlepaddle.github.io/PaddleX/latest/pipeline_deploy/edge_deploy.html
感谢您的解答，这个方法我们试了，我们无法借鉴实现方式，按照教程跑了一下还有报错，整体而言对我们帮助有限，请问还有什么解决办法吗，感谢。
